godel_scan_analysis/src/keyence_scan_server.cpp

```diff
@@ -0,0 +1,81 @@
+#include "godel_scan_analysis/keyence_scan_server.h"
+
+#include <pcl_ros/transforms.h>
+#include <pcl/filters/voxel_grid.h>
+
+// Constants
+const static double VOXEL_GRID_LEAF_SIZE = 0.005; // 5 mm
+
+const static double VOXEL_GRID_PUBLISH_PERIOD = 2.0; // seconds
```

Comment:
These properties are likely to undergo a tuning stage and so loading them as ros parameters will facilitate this.